common/settings.h

```diff
@@ -86,6 +86,10 @@ struct Settings {
   uint32_t observatory_port = 0;
   bool ipv6 = false;
 
+  // Determines whether an authentication code is required to communicate with
+  // the VM service.
+  bool disable_service_auth_codes = true;
```

Comment:
Can this not be a negative please? `enable_service_auth_codes = false`

Reply:
This is meant to be temporary and will be enabled by default later. I'd really rather avoid having to change everything only to have to change it back in the next day or so.